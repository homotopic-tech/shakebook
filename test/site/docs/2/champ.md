---
title: Hi I'm champ
description: Woof
---

Lorem ipsum
