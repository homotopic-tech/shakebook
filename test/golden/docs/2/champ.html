<html>
  <head>
    <title>Hi I’m champ</title>
  </head>
  <body>
    <h1>This post page is called Hi I’m champ</h1>
    <p>It was generated from the source docs/2/champ.md</p>
    <p>For the url /docs/2/champ.html</p>
    <p><p>Lorem ipsum</p></p>
    <p>Here are the subsections:
      <ul>
      </ul>
    </p>
  </body>
</html>
